View/TablePresets.php

```diff
@@ -13,6 +15,11 @@ public static function userSearch() {
                     $row = $row['first'] . ' ' . $row['last'] . '(' . $row['email'] . ')';
                 }
             ),
+            'display_value' => function($row, $fieldName) {
+                $user = Database::getInstance()->selectRow('user', ['user_id' => $row[$fieldName]], ['first', 'last', 'email']);
```

Comment:
this is going to cause it to query on each row - i think it would be better to just use the 'lookup' type for the field and just show the email address